src/App.css

```diff
@@ -154,3 +154,16 @@ body {
 .text-wrap-75 {
   max-width: 32em;
 }
+
+.full-width {
+  /* width: 70%; */
+}
+
+.progress-bar {
```

Comment:
this may need to be more specific so it doesn't affect other general progress bars later